### PR TITLE
Recreate homepage in css and add button solid option

### PR DESCRIPTION
**Pull request by sonia-y**

### CODE REVIEW

File: app/assets/javascripts/components/home_page.es6.jsx [hunk 1]
```diff
@@ -3,7 +3,51 @@ class HomePage extends React.Component {
     return (
       <div className="page-wrapper">
         <Header />
-          <div className="content-wrapper">
+          <div className="homepage content-wrapper">
+            <div className="title-wrapper">
+              <div className="title-text">
```
Comment (eric-ls) [inline app/assets/javascripts/components/home_page.es6.jsx]: nit: try using more semantic html like `<h1>` or `<p>`

File: app/assets/javascripts/components/home_page.es6.jsx [hunk 2]
```diff
@@ -3,7 +3,51 @@ class HomePage extends React.Component {
...
+                We connect underserved youth with experienced 
+                musicians to provide access to affordable music lessons.
+              </div>
+            </div>
+            <div className="instruments-wrapper">
+              <div className="instrument">
+                <img src={ImageConstants.instruments.piano} />
```
Comment (eric-ls) [inline app/assets/javascripts/components/home_page.es6.jsx]: nit: use `alt=""`, its necessary for accessibility (screenreaders)

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 3]
```diff
@@ -2,9 +2,9 @@ class ProgramPage extends React.Component {
 
   render () {
     return (
-      <div className="programpage page-wrapper">
+      <div className="page-wrapper">
         <Header />
-        <div className="content-wrapper">
+        <div className="programpage content-wrapper">
```
Comment (eric-ls) [inline app/assets/javascripts/components/program_page.es6.jsx]: nit: I would separate words with "-" so `program-page`

File: app/assets/stylesheets/pages/home.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+  .title-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: $orange60;
+    background-image: url(asset-path('stencils/instrument_white.png'));
+    width: 100vw;
+    background-size: 85vw auto;
```
Comment (eric-ls) [inline app/assets/stylesheets/pages/home.scss]: I don't have your app locally so I can't check myself, but when using percentage based things like `vw` and `vh`, make sure you resize your browser frequently to make sure things don't get weird on smaller or larger screens

**Comment by lilshim** [comment]
@sonia-y LGTM

**Pull request merged**